### DISCUSSION TITLE
Actions in forked repo?

### DISCUSSION THREAD

**Discussion by RusJJ** [community/community#50418] [Actions] [answered]
I'm kinda new to Actions so got in a little trouble with forked repository. It doesn't start the Actions and there's no button to do this. Does it mean i can't actually run Actions on it or it is disabled somewhere?

**Comment by airtower-luna** [comment, marked as answer]
As far as I know Actions are disabled on forks by default, to make sure workflows don't run under your user account before you have a chance to decide if you really want that. You should be able to enable them in the repository settings.

**Reply by RusJJ** [reply]
Sure, found it. Thanks. Also found i little typo in my yaml file.
